.travis.yml

```diff
@@ -1,25 +1,45 @@
 ---
 language: python
 
-python:
-    - 3.6
-    - 3.7
-
 env:
-    - CTAPIPE_VERSION="v0.7.0" CTAPIPE_IO_LST_VERSION="v0.4.1" PROTOZFITS_VERSION=v1.4.2
-    - CTAPIPE_VERSION="master" CTAPIPE_IO_LST_VERSION="master" PROTOZFITS_VERSION="master"
+  global:
+    - PYTHONIOENCODING=UTF8
+    - MPLBACKEND=Agg
+    - PYTEST_ADDOPTS='--color=yes'
+
 
 matrix:
+  include:
+    - name: "python=3.6, ctapipe=0.7"
+      python: 3.6
+      env:
+        - CTAPIPE_VERSION="v0.7.0"
+        - CTAPIPE_IO_LST_VERSION="v0.4.1"
+
+    - name: "python=3.7, ctapipe=0.7"
+      python: 3.7
+      env:
+        - CTAPIPE_VERSION="v0.7.0"
+        - CTAPIPE_IO_LST_VERSION="v0.4.1"
+
+    - name: "python=3.6, ctapipe=master"
+      python: 3.6
+      env:
+        - CTAPIPE_VERSION="master"
```

Comment:
do we really want to test against ctapipe `master`?

Reply:
This was what was there before. I think it is good. It is under allowed failures.

But this gives a nice heads up, that as soon as we are compatible to the next release, when we diverged from master again.

Reply:
ok, but it will start failing very soon after we are even with ctapipe master